admin/controller/dashboard/sale.php

```diff
@@ -30,29 +30,26 @@ public function index() {
             $data['percentage'] = 0;
         }
 
-        $sale_total = $this->model_report_sale->getTotalSales();
+        $config_currency = $this->config->get('config_currency');
+
+        $sale_total = $this->currency->format($this->model_report_sale->getTotalSales(), $config_currency, '', false);
 
         if ($sale_total > 1000000000000) {
-            $data['total'] = round($sale_total / 1000000000000, 1);
-            $suffix = 'T';
+            $data['total'] = round($sale_total / 1000000000000, 1) . $this->language->get('trillion_suffix');
         } elseif ($sale_total > 1000000000) {
-            $data['total'] = round($sale_total / 1000000000, 1);
-            $suffix = 'B';
+            $data['total'] = round($sale_total / 1000000000, 1) . $this->language->get('billion_suffix');
         } elseif ($sale_total > 1000000) {
-            $data['total'] = round($sale_total / 1000000, 1);
-            $suffix = 'M';
+            $data['total'] = round($sale_total / 1000000, 1) . $this->language->get('million_suffix');
         } elseif ($sale_total > 1000) {
-            $data['total'] = round($sale_total / 1000, 1);
-            $suffix = 'K';
+            $data['total'] = round($sale_total / 1000, 1) . $this->language->get('thousand_suffix');
         } else {
             $data['total'] = round($sale_total);
-            $suffix = '';
         }
 
-        $data['total'] = $this->currency->format($data['total'], $this->config->get('config_currency')) . $suffix;
+        $data['total'] = $config_currency . ' ' . $data['total'];
```

Comment:
Hm ... that's interesting, as some payment providers require display of currency code even if selling to only one country. Anyhow I thought all knew it, but if not, now they have the chance to finally learn it. ;) Not sure what you mean about the new variable though?